specify/specify6
# Begin Spanish Schema localization

- Reformat schema localization file
- Begin Spanish localization. Translated by:
   - Iñigo Granzow de la Cerda <igranzow@rjb.csic.es>
   - Mireia Beas

To test:
- Create a new database using the Spanish language in sp6 and see if any of the translated Spanish strings appears correctly on the form
